src/pages/console/uibuilder/slots.mdx

```diff
@@ -0,0 +1,150 @@
+};
+
+Slots allow you to nest other components as React code within your Studio UI components.  You can use slots to create dynamically generated children, like Comments on a Post, or to replace a child element altogether.
+
+## Adding a slot
+First, you'll need a component.  
+1. Log into Amplify and [navigate to Studio](/console/adminui/start/).
+
+2. On the left-hand navigation bar, click UI Library
+
+3. Select a Figma component from your UI Library that you've imported into Studio.  
+*If you don't have any Figma components already, you can start with [Amplify's Figma UI file](/console/uibuilder/figmatocode/).*
+
+4. Configure that component by clicking the Configure button in the upper right-hand corner.
+
+![Open your slot and click the gear icon to configure](/images/console/slots-console1.gif)
+
+Next, you'll add a slot to this component.  
+1. On the left-hand side, you'll see the Elements of your component.  Select a Figma Frame (![Figma frame icon](/images/console/figma-frame-icon.png)) within your component.  
+
+  In this example, the "Area" frame is selected.
+
+2. On the right-hand bar action bar, click the "Convert to a slot" button.  This will add a new component under the "Component properties" header.
+
+3. Optionally, change your your property name.  
+
+  In this example, the property has been renamed "comments"
+
+![Select your frame and add a slot](/images/console/slots-console2.gif)
+
+<Callout warning>
+
+Want to undo your slot creation?  
+
+Locate your slot in the Component properties, click the triple-dot icon, and click "Erase property" to remove the slot.
+
+</Callout>
+
+## Importing your component
+Once you've added a slot to your component, click the "Get component code" button at the bottom of the screen to see instructions on the next steps.  
+1. Copy the `amplify pull` command, and run it in your Terminal
+2. Copy the import code and paste it in your App.js file
+3. Lastly, render the component by calling it in JSX
+
+```jsx
+/*Import your component*/
+import { 
+  Ampligram 
+} from './ui-components'
+
+function App() {
+  return (
+    /*Add your component below*/
+    <div style={styles.post}>
+      <Ampligram  /> 
+    </div>
+  );
+}
+
+const styles = {
+  post: { width: 400, margin: '0 auto', padding: 20},
+}
+
+export default App;
+
+```
+
+Here's how the code above would render in your app.  Some minor styling has been added to help with visibility.
+
+![Imported component with a slot](/images/console/slots-base.png)
+
+## Using your slot
+To use the slot, pass a child component as a property of the parent component, using the prop name you picked earlier.  Then, the content you pass will be rendered as a child of the component.  
+
+
+````jsx
+import { 
+  Ampligram 
+} from '.ui-components'
+
+function App() {
+  return (
+    <div style={styles.post}>
+      <h1>Simple Slot Children</h1>
```

Comment:
I'm not sure we need the high-level div element. As long as the screenshots are "centered" I think we're good. The additional styles + elements might be distracting from the core of the guide.